rest/handler.go

```diff
@@ -441,29 +442,28 @@ func (h *handler) invoke(method handlerMethod, accessPermissions []Permission, r
 					}
 
 				} else {
-					return base.HTTPErrorf(http.StatusBadRequest, "Ambiguous keyspace: %s.%s", base.MD(keyspaceDb), base.MD(base.StringDefault(keyspaceScope, "")))
+					keyspaceScope = base.StringPtr(base.DefaultScope)
 				}
 			}
 			scope, foundScope := dbContext.Scopes[*keyspaceScope]
 			if !foundScope {
-				return base.HTTPErrorf(http.StatusNotFound, "keyspace %s.%s.%s not found", base.MD(keyspaceDb), base.MD(base.StringDefault(keyspaceScope, "")), base.MD(base.StringDefault(keyspaceCollection, "")))
+				return ksNotFound
 			}
 
 			if keyspaceCollection == nil {
 				if len(scope.Collections) > 1 {
-					// _default doesn't exist for a non-default scope - so make it a required element if it's ambiguous
-					return base.HTTPErrorf(http.StatusBadRequest, "Ambiguous keyspace: %s.%s", base.MD(keyspaceDb), base.MD(base.StringDefault(keyspaceScope, "")))
+					return base.HTTPErrorf(http.StatusNotFound, "keyspace %q not found", ks)
```

Comment:
`%q` -> `%s` (via `ksNotFound`)

```suggestion
					return ksNotFound
```